packages/core/test/unit/query-generator/get-constraint-snippet.test.ts

```diff
@@ -49,31 +50,31 @@ describe('QueryGenerator#_getConstraintSnippet', () => {
     it('generates a constraint snippet for a check constraint for a model', () => {
       const MyModel = sequelize.define('MyModel', {});
 
-      expectsql(() => queryGenerator._getConstraintSnippet(MyModel, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
+      expectsql(() => internals.getConstraintSnippet(MyModel, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
         default: 'CONSTRAINT [MyModels_age_ck] CHECK ([age] >= 10)',
         snowflake: checkNotSupportedError,
       });
     });
 
     it('generates a constraint snippet for a check constraint with schema', () => {
-      expectsql(() => queryGenerator._getConstraintSnippet({ tableName: 'myTable', schema: 'mySchema' }, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
+      expectsql(() => internals.getConstraintSnippet({ tableName: 'myTable', schema: 'mySchema' }, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
         default: 'CONSTRAINT [myTable_age_ck] CHECK ([age] >= 10)',
         snowflake: checkNotSupportedError,
       });
     });
 
     it('generates a constraint snippet for a check constraint with default schema', () => {
-      expectsql(() => queryGenerator._getConstraintSnippet({ tableName: 'myTable', schema: dialect.getDefaultSchema() }, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
+      expectsql(() => internals.getConstraintSnippet({ tableName: 'myTable', schema: dialect.getDefaultSchema() }, { type: 'CHECK', fields: ['age'], where: { age: { [Op.gte]: 10 } } }), {
         default: 'CONSTRAINT [myTable_age_ck] CHECK ([age] >= 10)',
         snowflake: checkNotSupportedError,
       });
     });
 
     it('generates a constraint snippet for a check constraint with globally set schema', () => {
       const sequelizeSchema = createSequelizeInstance({ schema: 'mySchema' });
-      const queryGeneratorSchema = sequelizeSchema.queryGenerator;
+      const queryGeneratorSchema = sequelizeSchema.queryGenerator.__TEST__getInternals();
```

Comment:
Nit; maybe something like `internalsSchema` might be clearer here

Reply:
Done!